web_src/js/components/RepoActionView.vue

```diff
@@ -503,8 +554,25 @@ export function ansiLogToHTML(line) {
   margin-left: 16px;
 }
 
-.job-step-container .job-step-summary:hover {
+.job-step-container .job-step-summary:hover,
+.job-step-container .job-step-summary.selected {
+  color: var(--color-console-fg);
   background-color: var(--color-black-light);
+  border-radius: 5px;
+}
+
+@media (max-width: 768px) {
+  .action-view-body {
+    flex-direction: column;
+  }
+  .action-view-left, .action-view-right {
+    width: 100%;
+  }
+
+  .action-view-left {
+    max-width: none;
+    overflow-y: hidden;
```

Comment:
`overflow` should be avoided if possible because it may prevent tooltips from rendering outside the box. Not an issue with tippy but still I would avoid. What issue is this trying to solve?

Reply:
`action-view-left` has a `overflow-y: scroll` on large screen, here on mobile if `overflow-y` is `scroll`, the width of `action-view-left` is not 100%

Reply:
Removed in https://github.com/go-gitea/gitea/pull/24718/commits/05330a1f697e47ecd20bea869a1225124229a7ce